src/app/ReadClient.cpp

```diff
@@ -422,6 +422,7 @@ CHIP_ERROR ReadClient::OnMessageReceived(Messaging::ExchangeContext * apExchange
     }
     else
     {
+        StatusResponse::Send(Protocols::InteractionModel::Status::InvalidAction, apExchangeContext, false /*aExpectResponse*/);
```

Comment:
This is still failing to send InvalidAction on various parsing errors for the message types we do expect, right?  Follow-up for that, please.

Reply:
updated, thanks